ipa-core/src/protocol/dp/insecure.rs

```diff
@@ -101,6 +101,63 @@ impl DiscreteDp {
     }
 }
 
+///  Non-negative DP noise for OPRF padding
+///  Samples from a Truncated Double Geometric
+#[derive(Debug)]
+pub struct OPRFPaddingDp {
```

Comment:
can you please also describe the functionality of it, i.e. how it samples and how it can be used to obtain the number of dummies to add (if that's what it is intended for)

Reply:
added documentation in dp/README.md